util/updateXML/update1.7.0.0.py

```diff
@@ -12,36 +12,96 @@
 # of simulation tools for power systems.
 from content.Ticket import ticket
 
-# rename IEEEG1 as IEEEG11, SCRX as SCRX1, IEEET1 : UStator0Pu -> Us0Pu, Standard.IEEET1_INIT -> Exciter_INIT
```

Comment:
The goal of updateXML files is to enable a smooth transition between two Dynawo releases. Therefore, what is the point of splitting this updateXML ?

Reply:
I have added lines for the name changes of the OpenModelica examples.

Reply:
> The goal of updateXML files is to enable a smooth transition between two Dynawo releases. Therefore, what is the point of splitting this updateXML ?

Yes, I agree but it just needs to be put in the order of how the merge was done.
Since we haven't changed anything in between the two developments, it makes sense to put them as you did, in the same folder. I think it won't change the final result
But it's more proper to have them separate to know what was done first.
I will ask to see if the order is important or not

Reply:
Regarding this : No need to separate that updateXML file. you can update the one of the ticket 3091.